dev/devicelab/bin/tasks/flutter_attach_test_android.dart

```diff
@@ -128,7 +128,7 @@ void main() {
         // If the next line fails, your device may not support regexp search.
         final String observatoryLine = await device.adb(<String>['logcat', '-e', 'Observatory listening on http:', '-m', '1', '-T', currentTime]);
         print('Found observatory line: $observatoryLine');
-        final String observatoryUri = RegExp('Observatory listening on ((http|\/\/)[a-zA-Z0-9:/=_\\-\.\\[\\]]+)').firstMatch(observatoryLine)[1];
+        final String observatoryUri = RegExp('Observatory listening on ((http|\\/\\/)[a-zA-Z0-9:/=_\\-\\.\\[\\]]+)').firstMatch(observatoryLine)[1];
```

Comment:
It looks like many of these might be easier to write with a raw string literal.

Reply:
agree. I will do the change.